examples/todomvc/index.js

```diff
@@ -5,12 +5,26 @@ import { Provider } from 'react-redux'
 import App from './containers/App'
 import configureStore from './store/configureStore'
 import 'todomvc-app-css/index.css'
+import { start, StateInfoConfig } from 'navigation'
+
+StateInfoConfig.build([
+    { key: 'todomvc', initial: 'app', states: [
+        { key: 'app', route: '{filter?}', defaults: { filter: 'all' }, trackCrumbTrail: false }
+    ]}   
+]);
```

Comment:
Builds the route configuration. There’s one route with an optional filter parameter that has a default value of 'all'